### DISCUSSION TITLE
Angular16 + primeflex 3.x : grid not working

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#599] [PrimeFlex]
Hello,

I created an Angular 16 client and also installed primeflex 3.1.1. Attached my angular.json, package.json. 
Then I tried using grid and pasted a sample from: [https://primeflex.org/gridsystem](url)
![Capture](https://github.com/primefaces/community/assets/4398228/20387945-2f23-4ef4-bb3e-05a976ed134f)

But I did not get the expected display. I got only:
6
6
12
4

[package.json](https://github.com/primefaces/community/files/13536013/package.json)
[angular.json](https://github.com/primefaces/community/files/13536066/angular.json)

Can you please advise ?

Thank you,
Zvika

**Comment by ThoSap** [comment]
Did you follow the installation/setup guide and import the required styles?
https://www.primefaces.org/primeng-v16/installation#styles

**Comment by ghost** [comment]
Hi Tomas,

Thank you for your reply.
After fixing the code:

*angular.json*
"styles": [

"node_modules/primeng/resources/themes/lara-light-blue/theme.css",
              "node_modules/primeng/resources/primeng.min.css",
              "node_modules/primeflex/primeflex.css",
              "src/styles.scss"
            ],

*styles.scss*
@import "primeng/resources/themes/lara-light-blue/theme.css";
@import "primeng/resources/primeng.css";

package.json
"primeflex": "^3.3.1",
"primeng": "^16.9.1",
But the grid is not working.

Best regards,
Zvika

**Comment by manshy099** [comment]
Hello every one , 

package.json
        "@angular/core": "^17.2.0",
        "primeflex": "^3.3.1",
        "primeng": "^17.10.0",


I am faceing the same Problem,
and I fount that the classes ( grid, col, col-1, etc... ) do not exist in "primeflex/themes/**primeone-light.css**" but the are exist in "primeflex/**primeflex.css**".

so I fixed that problem by improting the both files into style.scss

style.scss
@import "primeflex/themes/primeone-light.css";
@import "primeflex/primeflex.css";

Best regards.
Mishael

**Comment by ghost** [comment]
Hello,

In my project:
angular.json
  "dependencies": {
    ***@***.***/animations": "^17.3.0",
    ***@***.***/common": "^17.3.0",
    ***@***.***/compiler": "^17.3.0",
    ***@***.***/core": "^17.3.0",
    ***@***.***/forms": "^17.3.0",
    ***@***.***/platform-browser": "^17.3.0",
    ***@***.***/platform-browser-dynamic": "^17.3.0",
    ***@***.***/router": "^17.3.0",
    "primeflex": "^3.3.1",
    "primeng": "^17.12.0",
    "rxjs": "~7.8.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.14.3"
  },

styles.scss:
@import "primeflex/themes/primeone-light.css";
@import "primeflex/primeflex.css";

app.component.html:
<div class="p-grid p-dir-col">
  <div class="p-grid p-align-center p-nogutter">
    <label class="col1">Clock's Per Pri:</label>
    <input class="col2" [(ngModel)]="timingClkPri" type="text" pInputText>
  </div>
  <br/>
  <div class="p-grid p-align-center p-nogutter">
    <label class="col1">Pri Per Smc:</label>
    <input class="col2" [(ngModel)]="timingSmcPri" type="text" pInputText>
  </div>
</div>

app.component.scss:
.col1 {
    width: 50px;
    margin-left: 15px;
  }

.col2 {
    width: 50px;
    margin-left: 15px;
  }


But the controls are not aligned.
Can you please tell why ?

Thank you,
Zvika

**Reply by manshy099** [reply]
I did not get exactly what do you need
but to align items I used  Flexbox -> https://primeflex.org/flexdirection
give it a try.
